.circleci/config.yml

```diff
@@ -5,26 +5,9 @@
 version: 2
 jobs:
   build:
-    docker:
+    machine: true
```

Comment:
What is the main reason of using a machine and install docker?
For using docker-compose?

Reply:
Yes, for using docker-compose, also in circleci, if we use docker executor, the docker container that got spun up is remote, [so we can't use volume](https://support.circleci.com/hc/en-us/articles/360007324514-How-can-I-use-Docker-volume-mounting-on-CircleCI-), we need it to load our function table schema `foo.bar` into the cassandra.